scripts/style.sh

```diff
@@ -55,21 +55,24 @@ case "$version" in
     ;;
 esac
 
+# Ensure that tools in `Mintfile` are installed locally to avoid permissions
+# problems that would otherwise arise from the default of installing in
+# /usr/local.
+export MINT_PATH=Mint
+
+function swiftformat() {
```

Comment:
It looks like we only use this twice, and it may be less obvious if a developer goes to run this locally. Any objection to just changing the `swiftformat` to `mint run swiftformat` in the script below?

Reply:
No objection. Inlined.

Note that by hand use of `swiftformat` is not really something anyone can reasonably do. You need to pass a bunch of command-line options to control the style we use. Now you also need to set `MINT_PATH`. As a result, if you want to format a file without running all the checks you should just run `style.sh`.

Reply:
Yep that totally makes sense, I just realized that's exactly what I do.